src/SDKs/PolicyInsights/Management/PolicyInsights.Tests/PolicyTrackedResourcesTests.cs

```diff
@@ -0,0 +1,322 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace PolicyInsights.Tests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Microsoft.Azure.Management.PolicyInsights;
+    using Microsoft.Azure.Management.PolicyInsights.Models;
+    using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+    using Xunit;
+
+    public class PolicyTrackedResourcesTests
```

Comment:
I didn't do this in my test class (but I will).  Can you add a comment stating what service principal you used to record these tests (service principal ID and display name) so people can attempt to use it to re-record if necessary?

Reply:
Already deleted the SP, for obvious reasons :)

Reply:
besides, don't you need the SP secret to re-recored it?

Reply:
You would but it is a good starting point instead of creating a new SP for each record.  We shouldn't have to re-record these often but it can be a pain.  No problem if the SP is gone